gold-phone-input.html

```diff
@@ -261,7 +262,7 @@
     },
 
     ready: function() {
-      if (this.value && !Polymer.Element) {
+      if (this.value && !Polymer['Element']) {
```

Comment:
Not clear why this change is needed, is it for closure? `paper-input` uses the same [check here](https://github.com/PolymerElements/paper-input/blob/master/paper-input.html#L278)

Reply:
Phone input does the [same](https://github.com/PolymerElements/gold-phone-input/pull/92/files/27630ce88631f955a2f6a1691b72101c5079b7d3#diff-c3bd302f62769d8658c451f4aac6f428R253) but the compiler does not complain there. /shrug

Reply:
I meant it wasn't clear why the change from `!Polymer.element` to `Polymer['Element']`